src/tests/ftest/util/test_utils_pool.py

```diff
@@ -467,8 +467,16 @@ def rebuild_complete(self):
             bool: True if pool rebuild is complete; False otherwise
 
         """
-        self.display_pool_rebuild_status()
-        return self.info.pi_rebuild_st.rs_done == 1
+        status = False
+        if self.control_method.value == self.USE_API:
+            self.display_pool_rebuild_status()
+            status = self.info.pi_rebuild_st.rs_done == 1
+        elif self.control_method.value == self.USE_DMG and self.dmg:
+            self.set_query_data()
+            self.log.info(
+                "Pool %s query data: %s\n", self.uuid, self.query_data)
+            status = self.query_data["rebuild"]["status"] != "busy"
+        return status
```

Comment:
How about checking these 2 scenarios as well?
`if self.control_method.value == self.USE_DMG`, which can be an elif after line 478
and 
last when none of them are true which can be the final `else` statement
Check destroy() method under TestPool class above for example.